addons/courierEditor/particlePlacement.gd

```diff
@@ -0,0 +1,20 @@
+extends GPUParticles2D
+
+var frameSkip = 0
+
+# Called when the node enters the scene tree for the first time.
+func _ready():
+	pass # Replace with function body.
+	
+# Called every frame. 'delta' is the elapsed time since the previous frame.
+func _process(delta):
+	var parentSprite = get_parent()
+	var invert_extend_vec = (parentSprite.extend_vec * -1)	
+	var translateToParent = get_parent().position.direction_to(position) * -1
+	var distToParent = get_parent().position.distance_to(position)
+	var desiredTranslatePosition = translateToParent * distToParent
```

Comment:
These 3 is supposed to be how you need to move to be at your parent's position?
I'm pretty sure this can just be `get_parent().position - position` (or maybe the reverse, I sometimes get confused with vectors)

Reply:
That does seem to work, likely I did it in a more complicated than needed way

Reply:
changed in latest